.circleci/config.yml

```diff
@@ -11,34 +21,48 @@ workflows:
       - codacy/checkout_and_version:
           write_sbt_version: true
       - codacy/sbt:
-          name: publish_docker_local
+          name: test_and_coverage
+          persist_to_workspace: true
           cmd: |
-            sbt "set scalafmtUseIvy in ThisBuild := false;
+            export NVM_DIR="/opt/circleci/.nvm"
+            [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+            nvm install v8.11.3
+            nvm alias default v8.11.3
+
+            npm install
+            export PATH=$PATH:$(pwd)/node_modules/.bin
+            STYLELINT_CONFIG_BASEDIR=$(pwd)/node_modules sbt "set scalafmtUseIvy in ThisBuild := false;
```

Comment:
Required because for testing the tool using sbt test as it needs node to be installed as well as stylelint dependencies